### ISSUE TITLE
[BUG] Received few bytes!

### ISSUE THREAD

**Issue by BloodDragon2580** [KOOKIIEStudios/PalCON-Discord#15]
Describe the bug

When i use the command "/announce" the first time it works, but the second time I get the error in the console:
"ERROR - Unable to make game announcement: Received few bytes!"

```
The first text was: "Um 20 Uhr ist der"
The second text was: "für 10min Offline"
```

The first text worked, the second didn't.

```
2024-02-03 19:41:33,509 - client - INFO - Setting up RCON connection
2024-02-03 19:41:33,513 - client - DEBUG - Broadcasting message to world
2024-02-03 19:41:33,643 - __main__ - ERROR - Unable to make game announcement: Received few bytes!
```

Desktop (please complete the following information):
**- OS: Windows 2019

Version: Latest Build** https://github.com/KOOKIIEStudios/PalCON-Discord/commit/88d0285ed0aa83f0d87d283924e0b0ac205a6480

**Comment by Bratah123** [comment]
Duplicate Issue with #6 

TLDR; rcon client we are using is only assuming that latin-characters are coming in from the server, see issue thread for what we are trying to do.

**Comment by BloodDragon2580** [comment]
oh sry for duplicatet i dont have see it. Sry.
